documents/document_store.go

```diff
@@ -12,8 +13,10 @@ type IDocumentStore interface{
 
 type DocumentStore struct{
 	Urls []string
-	Database string
+	Database, ApiKey string
```

Comment:
We don't have API keys anymore. Please remove references to those. See: https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Client/Documents/DocumentStore.cs